workflow/scripts/rename_contigs.py

```
@@ -0,0 +1,15 @@
+import pysam
+
+sys.stderr = open(snakemake.log[0], "w")
```

Ruff (0.8.0)

Undefined name `sys`

(F821)